magento/magento2
# Adding Extra abilities modules.

### Description (*)
Extra abilities module: added Extra abilities customer extension attribute and extra comments product extension attribute.


### Manual testing scenarios (*)

1. Create a customer from frontend
2. Go to the customer edit page -->Account Information --> Is Allowed add extra comemnt --> Yes
3. Go to the product page --> Extra comments tab 
4. Send some comment
5. Admin --> Extra abilities --> Extra Comments --> Approve your comment
6. Check on the product page or you can get product from api
